packages/tempus/src/Tempus_StepperExplicit_impl.hpp

```diff
@@ -0,0 +1,165 @@
+// @HEADER
+// ****************************************************************************
+//                Tempus: Copyright (2017) Sandia Corporation
+//
+// Distributed under BSD 3-clause license (See accompanying file Copyright.txt)
+// ****************************************************************************
+// @HEADER
+
+#ifndef Tempus_StepperExplicit_impl_hpp
+#define Tempus_StepperExplicit_impl_hpp
+
+
+namespace Tempus {
+
+
+template<class Scalar>
+void StepperExplicit<Scalar>::setModel(
+  const Teuchos::RCP<const Thyra::ModelEvaluator<Scalar> >& appModel)
+{
+  this->validExplicitODE(appModel);
+  this->appModel_ = appModel;
+}
+
+
+template<class Scalar>
+void StepperExplicit<Scalar>::setNonConstModel(
+  const Teuchos::RCP<Thyra::ModelEvaluator<Scalar> >& appModel)
+{
+  this->setModel(appModel);
+}
+
+template<class Scalar>
+void StepperExplicit<Scalar>::setSolver(std::string solverName)
```

Comment:
Does this mean that explicit stepper can now overlook the solver PL if it exists?

Reply:
I think the answer is yes.  If not we should make it work that way.

Reply:
I just tested this in Drekar. Tempus did not ignore the solver name or the solver PL. 

```
435: p=1 | *********** Caught Exception std::exception: Begin Error Report ***********¬
435: p=1 | Error, the parameter {name="Default Solver",type="ParameterList",value="..."}¬
435: p=1 | in the parameter (sub)list "Drekar Parameters->Tempus->Euler Example"¬
435: p=1 | was not found in the list of valid parameters!¬
435: p=1 |·¬
435: p=1 | The valid parameters and types are:¬
435: p=1 |   {¬
435: p=1 |     "Use FSAL" : bool = 0¬
435: p=1 |     "Initial Condition Consistency" : string = None¬
435: p=1 |     "Initial Condition Consistency Check" : bool = 1¬
435: p=1 |     "Use Embedded" : bool = 0¬
435: p=1 |     "Description" : string = RK Explicit 2 Stage 2nd order by Runge¬
435: p=1 | Also known as Explicit Midpoint¬
435: p=1 | Solving Ordinary Differential Equations I:¬
435: p=1 | Nonstiff Problems, 2nd Revised Edition¬
435: p=1 | E. Hairer, S.P. Norsett, G. Wanner¬
435: p=1 | Table 1.1, pg 135¬
435: p=1 | c = [  0  1/2 ]'¬
435: p=1 | A = [  0      ]¬
435: p=1 |     [ 1/2  0  ]¬
435: p=1 | b = [  0   1  ]'¬
435: p=1 |·¬
435: p=1 |     "Stepper Type" : string = RK Explicit 2 Stage 2nd order by Runge¬
435: p=1 |   }¬
435: p=1 |·¬
435: p=1 |·¬
435: p=1 | Throw number = 1¬
```

It's not an issue with this PR but I think it's somethign we should definitely consider

Reply:
Yeah, let talk this through and create another issue.